test/models/local_restriction_test.rb

```diff
@@ -0,0 +1,69 @@
+describe LocalRestriction do
+  let(:restriction) { described_class.new("E08000001") }
+
+  it "returns the area name" do
+    LocalRestriction.stub_const(:FILE_PATH, "test/fixtures/local-restrictions.yaml") do
```

Comment:
I did a little bit a research into stubbing constants in minitest too, and I found the same `minitest-stub-const` gem too.

Is it possible to wrap all of the tests in the block?

i.e.

```
describe LocalRestriction do
  LocalRestriction.stub_const(:FILE_PATH, "test/fixtures/local-restrictions.yaml") do
    let(:restriction) { described_class.new("E08000001") }

    it "returns the area name" do
      assert_equal "Tatooine", restriction.area_name
    end
.
.
.
```

Reply:
Wasn't able to get that working. For some reason the code doesn't fail but just uses the prod data :/ I wasn't happy with the amount of times that I had to call the `minitest-stub-const` gem either

Reply:
That's so annoying!

Reply:
yeah, agreed :/